### DISCUSSION TITLE
【委托】我朋友生病了！

### DISCUSSION THREAD

**Discussion by Soldier11-ObsidianBarracks** [share121/inter-knot#3335] [其他]
我的朋友生病了，由于防卫军的工资还没发下来，我只能找你们资助我了，需要328元

**Comment by clcl-maker** [comment]
你朋友不会是大闹天宫的那位吧（doge

**Reply by Soldier11-ObsidianBarracks** [reply]
呃（哈哈）

**Comment by icenow188** [comment]
这个委托我接了！
把你的手机给我<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >
![OIP-C](https://github.com/user-attachments/assets/1833435d-1cf3-48fd-b85d-330be12e9a42)

**Comment by Bahajsjsh** [comment]
还挺贪心要豪华版
